queries.sql

```diff
@@ -1,3 +1,23 @@
-/*Queries that provide answers to the questions from all projects.*/
+SELECT * FROM animals
+WHERE name LIKE '%mon';
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals
+WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+
+SELECT * FROM animals
+WHERE neutered = true AND escape_attempts <3;
+
+SELECT date_of_birth FROM animals
+WHERE name IN ('Agumon', 'Pikachu');
```

Comment:
- [ ] Greate job so far 👏🏻 However, According to the project requirements, you should add a screenshot of their query results to the pull request description and not any link to the google docs. This also helps to ensure that you run the queries with success. Please make this 👍🏻